History.md

```diff
@@ -1,25 +0,0 @@
-
-1.0.0 / 2014-05-05
```

Comment:
I kept it, just renamed is as `CHANGELOG`. It would need to be updated if this PR is merged thought.